pkg/api/steve/supportconfigs/generator_test.go

```diff
@@ -0,0 +1,182 @@
+// unit tests for generator
+package supportconfigs
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/rancher/rancher/pkg/managedcharts/cspadapter"
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+type FakeChartUtil struct {
+	adapterUtil      *cspadapter.ChartUtil
+	generateError    bool // return a random error
+	generateNotFound bool // return cspadapter.ErrNotFound
+}
+
+func NewFakeChartUtil(generateError bool, generateNotFound bool) *FakeChartUtil {
+	return &FakeChartUtil{
+		generateError:    generateError,
+		generateNotFound: generateNotFound,
+	}
+}
+
+func (c *FakeChartUtil) GetRelease(chartNamespace string, chartName string) (*release.Release, error) {
```

Comment:
Please replace any unused parameter names with `_` in this file - else our linter might complain and fail CI.

Reply:
Will do.